client/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+  <head>
+    <title> TO DO APP </title>
+  </head>
+  <body>
+    <h1> Hi I'm working </h1>
```

Comment:
## Critical

So the application isn't working yet, what you should have here should be the `div` we are targeting. Take a look at this: [https://github.com/BolajiOlajide/Webpack3-React-StarterKit/blob/master/src/index.html#L7](https://github.com/BolajiOlajide/Webpack3-React-StarterKit/blob/master/src/index.html#L7).

Reply:
If you don't understand what I mean, let me know so we can look into it.